src2/App.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react'
+import './App.css'
+import 'bootstrap/dist/css/bootstrap.css'
+import 'font-awesome/css/font-awesome.css'
+import AddNew from './components/AddNew/AddNew'
+
+export default class App extends Component {
```

Comment:
Nice: Use a more telling name like
```suggestion
export default class ToDoApp extends Component {
```